content/news/news.20240422.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Taskwarrior 3.0.2 Released"
+date: 2024-04-22
+---
+
+### Taskwarrior 3.0.2 Released
+
+This patch release to Taskwarrior 3.0.0 addresses a few minor issues in the 3.0.0 release: improvements to task news, better error handling and documentation, and fixes for interactions with hooks. It also fixes an issue with the release tarball, #3294.
```

Comment:
I wonder whether `#3294` will link to https://github.com/GothenburgBitFactory/taskwarrior/issues/3294/ 🤔